### PR TITLE
add some tests

### PR DESCRIPTION
**Pull request by fphammerle**

### CODE REVIEW

**Comment by fphammerle** [comment]
TODO: test `import namespace.module`

started here: https://github.com/fphammerle/pylint-import-requirements/commit/61b70588ad63068a4eaa258b048c29deb2551932

however, I need to workaround this check:
```
if "site-packages" not in origin_path:
    return
```
https://github.com/WanzenBug/pylint-import-requirements/blob/e57ecb530e7bff17eaf32b513c3cb284b8770a18/pylint_import_requirements/__init__.py#L148

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+# required for python >= 3.7
+dist: xenial
+
+install:
+- pip install pipenv
+- pipenv sync --dev
+- pipenv graph
+
+script:
+- pipenv run pytest --cov=pylint_import_requirements --cov-report=term-missing
```
Comment (WanzenBug) [inline .travis.yml:19]: also run pylint itself

File: tests/test_pylint_command.py [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import subprocess
+
+
+def _run_pylint(package_dir_path: pathlib.Path) -> subprocess.CompletedProcess:
+    return subprocess.run(
+        ('pylint', '--score=n', '--load-plugins=pylint_import_requirements', 'module.py'),
+        cwd=package_dir_path,
+        stdout=subprocess.PIPE,
+        check=False,
+    )
```
Comment (WanzenBug) [inline tests/test_pylint_command.py]: Have you checked out [this document](http://pylint.pycqa.org/en/latest/how_tos/custom_checkers.html#testing-a-checker)? Its seems to be an easier solution than manually running pylint and asserting the output

**Review by WanzenBug — CHANGES_REQUESTED** [review]

File: tests/test_pylint_command.py (hunk 2 above)
Comment (fphammerle) [reply tests/test_pylint_command.py]: Thanks for the review and the suggestions! :-)

I didn't check the docs :joy:

Did you expect something like this https://github.com/WanzenBug/pylint-import-requirements/blob/2a9fc2830ccbb70da3c6368d64cc2660f2b831a5/tests/test_checker.py ?

Shall we remove the subprocess.run integration tests completely or keep one?

Comment (WanzenBug) [reply tests/test_pylint_command.py]: I would prefer complete removal.

**Review by WanzenBug — APPROVED** [review]

**Pull request merged**